ficlatte/settings-example.py

```diff
@@ -1,106 +0,0 @@
-"""
-Django settings for ficlatte project.
```

Comment:
You didn't add the new file to this commit, so it's showing only that you deleted `settings-example.py`.